draft-ietf-scitt-architecture.md

```diff
@@ -123,14 +123,12 @@ informative:
 
 --- abstract
 
-Traceability of physical and digital Artifacts in supply chains is a long-standing, but increasingly serious security concern.
+Traceability of physical and digital artifacts in supply chains is a long-standing, but increasingly serious security concern.
```

Comment:
Artifact is defined in terminology, which we've said we want to keep Capatilized.

Reply:
Exactly.
But a) this this the abstract and b) how do we deal with usage of capitalized terms in the text before they are introduced in the terminology Section?